script.js

```diff
@@ -0,0 +1,8 @@
+var list = document.getElementById('list'),
+	add = document.getElementById('addElem');
+
+add.addEventListener('click', function() {
+	var element = document.createElement('li');
+	element.innerHTML = 'item' + ' ' + (document.getElementsByTagName('li').length);
```

Comment:
why not `list.children.length` - you declare before `list` so you can use it also in this case